cmd/nitro/nitro.go

```diff
@@ -581,3 +596,59 @@ func applyArbitrumAnytrustGoerliTestnetParameters(k *koanf.Koanf) error {
 		"persistent.chain": "goerli-anytrust",
 	}, "."), nil)
 }
+
+func testIndexUpdated(chainDb ethdb.Database, lastBlock uint64) bool {
```

Comment:
Maybe rename this to `testTxIndexUpdated` to clarify which index it's operating on